### PR TITLE
Create install dir on the fly

### PR DESCRIPTION
**Pull request by DonavanFritz**
In this PR, we fix the assumption that the collector install dir `/usr/local/logicmonitor` already exists before downloading the collector binary by creating the dir(s) if it does not already exist.

### CODE REVIEW

File: logicmonitor_core/Collector.py [hunk 1]
```diff
@@ -85,6 +85,10 @@ def get_installer_binary(self):
             logging.debug("Platform is Linux")
             logging.debug("Agent ID is " + str(self.id))
 
+            if not os.path.exists(self.installdir):
+                os.makedirs(self.installdir)
+                logging.debug("Created installdir at `{dir}`".format(dir=self.installdir))
```
Comment (woz5999) [inline logicmonitor_core/Collector.py]: Please wrap this line to <80 chars and use + for string concat instead of .format

**Comment by woz5999** [comment]
Thanks for the PR Donovan. I left a code comment with 2 minor style fixes we'd like to see before merging.

**Comment by DonavanFritz** [comment]
woz5999-
I have updated the PR with the suggested style fixes.

**Comment by woz5999** [comment]
@DonavanFritz code changes merged. New version released to pypi.

**Pull request merged**